### ISSUE TITLE
[enterprise-4.7] Issue in file cli_reference/openshift_cli/getting-started-cli.adoc

### ISSUE THREAD

**Issue by mgoerens** [openshift/openshift-docs#33025]
### Which section(s) is the issue in?

Section `Installing the OpenShift CLI` / `Installing the OpenShift CLI by downloading the binary` for all three Operating Systems.

### What needs fixing?

The procedure described doesn't allow the user to download the OpenShift CLI.

In particular, in step 1, the "Infrastructure Provider" page which is referenced is https://cloud.redhat.com/openshift/install. This is actually redirecting to https://cloud.redhat.com/openshift/create. The rest of the procedure, especially Step 2 & 3, doesn't correspond to this page.

As a workaround, I was able to download the CLI using this mirror: https://mirror.openshift.com/pub/openshift-v4/clients/ocp/

**Comment by lbscorpio** [comment]
@mgoerens I think no issue here. The URL can be redirected. The important step is Step 2, to select your infrastructure provider(e.g. cloud provider AWS or datacenter infrastructure provider OpenStack) correctly. Once step 2 done, then can see the section of Step 3.

Below screenshot was captured by choosing the infrastructure provider "Bare Metal", fyi, thanks.
<img width="720" alt="oc-cli" src="https://user-images.githubusercontent.com/46341043/120915544-ce37da80-c6d6-11eb-9d96-23810b202397.png">

**Comment by mgoerens** [comment]
You are right. Contrary to what I wrote originally, it is indeed possible to download the CLI from this webpage, and the procedure is correct.

As a user, however, I find this procedure confusing. I was expecting a simple download button directly on the target page, and not to go through several options and menus to find the download button. That's why I thought something is wrong with the doc and decided to open this issue.

There is actually no added value in selecting the infrastructure provider and installation type. AFAIK, the OpenShift client for Linux is the same in all cases. I believe it should be straightforward to download the OpenShift client.

Why not use the mirror link I posted earlier directly in the OpenShift docs and some command line instructions? For instance:

```
wget https://mirror.openshift.com/pub/openshift-v4/clients/ocp/latest/openshift-client-linux.tar.gz
tar -zxvf openshift-client-linux.tar.gz
mv oc kubectl /usr/local/bin/
rm openshift-client-linux.tar.gz 
```

**Comment by jboxman** [comment]
@mgoerens, I'm not sure that's signed for OS X? I forget which place to get the binary isn't signed, but there was one.

**Comment by mgoerens** [comment]
@jboxman Sorry I'm not familiar with OS X environments and can't answer your question.

**Comment by jboxman** [comment]
Refs https://github.com/openshift/openshift-docs/issues/19589.

**Comment by lbscorpio** [comment]
@mgoerens If using the mirror link, the OpenShift docs have to update the link on every major release GA or url path changed(e.g. openshift-v4 change to openshift-v5). And there are different ways to get the `oc` binary(website, cli, RPM), I think we can't remove the way from website on instruction and only leave the CLI method. Of course, docs team can mention the mirror URL on the docs page as a workaround. Fyi, thanks.
